tex/appendix/ans-fo-semantics.tex

```diff
@@ -179,6 +179,24 @@ \subsection*{9.7.6}
     \emptyset$ is valid in our model.
 \end{enumerate}
 
+\subsection*{9.7.10}
+
+\begin{enumerate}
+    \item[i] We will prove $\mathcal{M} \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$ iff $P^{\mathcal{M}}$ has at least two elements. \\\\
+    Left-to right: We will prove this using proof by contradiction. Let $\mathcal{M}$ and  $\alpha$ be arbitrary such that $\mathcal{M}, \alpha \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. Assume $P^\mathcal{M}$ has at most 1 element. We can distinguish two cases: $P^\mathcal{M}$ has 0 elements (1) and $P^\mathcal{M}$ has 1 element (2). \\ 
+    (1) We assume $P^\mathcal{M} = \emptyset$. Then there is no $d \in D^\mathcal{M}$ such that $P(d)$ is true. This means that the conjunction $P(x) \wedge P(y) \wedge x \neq y$ can never be true. But we assumed $\mathcal{M}, \alpha \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. Thus, we have a contradiction. It follows that if $P^\mathcal{M} = \emptyset$, $\mathcal{M}, \alpha \nvDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. \\
```

Comment:
It's better not to say that $P(d)$ is true (since $P$ is predicate and $d$ an element of the domain). Better say: there is no $d$ such that $d\in P^\mathcal{M}$. 
You can then argue that this means that for all $d\in D^\mathcal{M}$ and assignment $\alpha$ that $\mathcal{M},\alpha[x\mapsto d]\nvDash P(x)$. At the same time, you could argue that there has to be a $d\in D^\mathcal{M}$ such that $\mathcal{M},\alpha[x\mapsto d]\vDash P(x)$ since $\mathcal{M}\vDash \exists x\exists y (P(x)\land P(y)\land x\neq y)$ by assumption.